napi.h

```diff
@@ -355,6 +357,20 @@ namespace Napi {
     /// be needed to store this BigInt (i.e. the return value of `WordCount()`).
     void ToWords(int* sign_bit, size_t* word_count, uint64_t* words);
   };
+
+  /// A JavaScript date value.
+  class Date : public Value {
+  public:
+    static Date New(
+      napi_env env, ///< N-API environment
+      double value ///< Number value
+    );
+
+    Date();                               ///< Creates a new _empty_ Date instance.
+    Date(napi_env env, napi_value value); ///< Wraps a N-API value primitive.
+
+    double NumberValue() const;   ///< Converts a Date value to a number value.
```

Comment:
Please substitute  `///< Converts a Date value to a number value.` with `///< Converts a Date value to a double primitive.`